config/jest.setup.js

```diff
@@ -37,24 +40,24 @@ global.window.__scalprum__ = {
       module: 'inventory#./RootApp',
       name: 'inventory',
     },
-  },
-  factories: {
-    inventory: {
-      expiration: new Date('01-01-3000'),
-      modules: {
-        './InventoryTable': {
-          __esModule: true,
-          default: () => (
-            <div>
-              <h1>Inventory mock</h1>
-            </div>
-          ),
-        },
-      },
+    remediations: {
+      manifestLocation: 'https://console.stage.redhat.com/apps/remediations/fed-mods.json?ts=1643875037626',
+      module: 'remediations#./RemediationButton',
+      name: 'remediations',
     },
   },
 };
 
+jest.mock('@redhat-cloud-services/frontend-components/Inventory', () => {
+  return {
+    InventoryTable: () => (
+      <div>
+        <h1>Inventory mock</h1>
+      </div>
+    ),
+  };
+});
+
```

Comment:
These mocks are global to all tests right?

Reply:
Yep. IDK if it is possible to override it somehow in the individual tests because it somehow runs this `jest.setup.js` file before every test file, but now it is not problem :+1:

Also this mock for inventory table is not ideal, what I see in other insights projects they have some scalprum config mockup, but I couldn't get it to work :shrug: